src/worldmap.ts

```diff
@@ -101,11 +104,20 @@ export default class WorldMap {
     return !_.isEqual(locations, dataPoints);
   }
 
+  // Determines if the log scale is applied and chooses which value to use
+  // Used when calculating circle sizes
+  getValue(dataPoint) {
+    return parseInt(this.ctrl.panel.isLogScale ? Math[this.ctrl.panel.logFunction](dataPoint.value) : dataPoint.value)
```

Comment:
This expression looks super ugly :)

I do believe if you split this into multiple lines it would greatly improve the legibility.

Reply:
I DO agree! :)